### PR TITLE
Allow files in same directory to be loaded without prefixing './'

### PR DESCRIPTION
**Pull request by ndwarshuis**

### CODE REVIEW

File: dhall/__init__.py [hunk 1]
```diff
@@ -16,5 +16,7 @@ def remember_cwd():
 
 def load(fp):
     with remember_cwd():
-        os.chdir(os.path.dirname(fp.name))
+        newdir = os.path.dirname(fp.name)
+        if newdir != "":
+            os.chdir()
```
Comment (s-zeng) [inline dhall/__init__.py:21]: Looks like you just need to put in the newdir argument and you should be good.

**Review by s-zeng — CHANGES_REQUESTED** [review]
Thanks for contributing! Let's get the test passing again, and maybe add a test as well, and we should be good to go

**Comment by ndwarshuis** [comment]
Fixed typo and also added a new test (and the tests should actually pass). I just reused the already-present test file for the new test and called it in a slightly different order; hopefully that's clear enough.

**Comment by ndwarshuis** [comment]
@s-zeng were you still waiting on me to do something more with this?

**Review by s-zeng — APPROVED** [review]

**Comment by ndwarshuis** [comment]
thanks :)

**Pull request merged**